src/main/java/frc/team166/robot/RobotMap.java

```diff
@@ -74,12 +76,19 @@
         public static final String K_I = "kI";
         public static final String K_D = "kD";
         public static final String K_F = "kF";
+        public static final String K_PATRICK = "kPatrick";
```

Comment:
Let's name this K_P_Manipulator... And apply that the the others ones too...

Reply:
Why do you have to ruin all of my fun?